src/ListItem/ListItem.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './ListItem.css';
+
+const ListItem =  ({ label, body })  => {
+
+  const formatBody = () => {
+    if (body === '$0') {
+      return '-'
+    }
+    return body;
+  }
+
+  return (
+    <section className='details-box'>
+      <p className='l-details'><b>{label}</b></p>
```

Comment:
Looks like you might only be using this class once. You could replace the `<b>` tag with a css rule.